convert-pth-to-ggml.py

```diff
@@ -123,19 +177,30 @@ def get_n_parts(dim):
             fout.write(struct.pack("i", len(text)))
             fout.write(text)
 
-    for k, v in model.items():
+    model = load_model(fname_model)
+
+    q = queue.Queue(maxsize=2)
+
+    def writer():
+        while True:
```

Comment:
`while True`? Does this function ever return? I don't know if the function exists but maybe something like `while !q.atEnd()`

Please correct me if I'm wrong. I haven't worked with Python since a year or so.

Reply:
fixed!